internal/provider/nordvpn/updater/models.go

```diff
@@ -47,6 +47,8 @@ type serverData struct {
 		} `json:"metadata"`
 	} `json:"technologies"`
 	Groups []struct {
+		// Identifier can be 'legacy_standard', 'legacy_p2p'.
+		Identifier string `json:"identifier"`
```

Comment:
If the group can only be `legacy_standard` or `legacy_p2p`, would it be reasonable instead to just set the `PortForward` server boolean when encountering `legacy_p2p` and not have categories at all?

Reply:
Just double checked and turns out I missed `legacy_dedicated_ip`
```
"legacy_dedicated_ip"
"legacy_p2p"
"legacy_standard"
```

NGL I'm not sure what you mean about the PortForward bit, what relation to this does it have?

Side note: NordVPN does not support port forwarding

Reply:
Yes my bad I may had mixed with another provider.
Another question: wasn't the server groups meant to be for example `Double_VPN`, `Africa_The_Middle_East_And_India`, etc. as mentioned in https://github.com/qdm12/gluetun/issues/1541#issue-1680084182 ?